src/main/resources/com/ncc/aif/ontologies/InterchangeOntology

```diff
@@ -245,6 +245,13 @@
   rdfs:range xsd:string ;
   rdfs:subPropertyOf owl:topDataProperty .
 
+:sourceDocument rdf:type owl:DatatypeProperty ;
+  rdfs:label "source document" ;
+  rdfs:comment "Specify xsd:string identifier for source document of justification" ;
```

Comment:
Suggest "[...] source element of justification" for `aida:source` and possibly "[...] source parent document of justification" for `aida:sourceDocument`.

Reply:
fixed